docs/conf.py

```diff
@@ -14,7 +14,9 @@
 import sys
 import subprocess
 
-sys.path.insert(0, os.path.abspath('.'))
```

Comment:
Since `.` was added to `sys.path` via `conf.py`, and in `docs` the folder `new_doc` was properly renamed to `hoc`, we would get a clash when importing neuron(that is `neuron.hoc`).

Reply:
I wonder why that was there, but the only other Python file in the folder is parse_rst, which isn't used here, so I think this is fine.

Reply:
I added it at the consolidation phase way back, when retrofitting a single `html2.py` for both `progref-py` and `progref-hoc`.